### PR TITLE
Add external account verify

### PR DESCRIPTION
**Pull request by wbronitsky**
r? @stripe/api-libraries

### CODE REVIEW

File: src/main/java/com/stripe/model/ExternalAccount.java [hunk 1]
```diff
@@ -77,4 +91,8 @@ public DeletedExternalAccount delete(RequestOptions options) throws
 			APIConnectionException, CardException, APIException {
 		return request(RequestMethod.DELETE, this.getInstanceURL(), null, DeletedExternalAccount.class, options);
 	}
+
+	private String getValidateURL() {
```
Comment (brandur) [inline src/main/java/com/stripe/model/ExternalAccount.java]: Kind of a nit: but putting the URL formatting in a new method like this seems a little unconventional relative to the rest of the library. What do you think about putting the `String.Format` [in-line as seen here](https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Invoice.java#L387,L388)?

**Comment by brandur** [comment]
+1 other than the minor comment above and that it seems like your editor may have had a whitespace failure here (check the diff in GitHub if it's not obvious locally). Looks good to come in after those are fixed up. Thanks!

**Comment by wbronitsky** [comment]
@brandur thanks for the review! the test and the spacing should be fixed

File: src/main/java/com/stripe/model/ExternalAccount.java [hunk 2]
```diff
@@ -54,6 +54,22 @@ public String getInstanceURL() {
 		}
 	}
 
+	public ExternalAccount verify(Map<String, Object> params) throws 
+			AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return verify(params, (RequestOptions) null);
+  }
```
Comment (brandur) [inline src/main/java/com/stripe/model/ExternalAccount.java]: Looks like the indent on this line could still be fixed.

File: src/test/java/com/stripe/model/ExternalAccountTest.java [hunk 3]
```diff
@@ -59,4 +59,26 @@ public void testUnknownExternalAccountDeletion() throws StripeException, IOExcep
 			null, APIResource.RequestType.NORMAL, RequestOptions.getDefault()
 		);
 	}
+
+	@Test
+  public void testVerify() throws StripeException, IOException {
```
Comment (brandur) [inline src/test/java/com/stripe/model/ExternalAccountTest.java]: And lastly, mind just converting the leading whitespace in this test to tabs as well? Thanks!

**Comment by wbronitsky** [comment]
@brandur sorry for firing blindly, I don't Java so well. Let me know if I need to change more

**Comment by brandur** [comment]
@wbronitsky NP. Thanks for fixing it up!

**Comment by brandur** [comment]
Released as 1.39.0.

**Pull request merged**